server/tests/routes/userRoute.spec.js

```diff
@@ -0,0 +1,54 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import app from "../../index";
+import database from "../../models/index";
+import userData from "../fixtures/model/userData";
+import { hashPassword } from "../../helpers";
+import { validateData } from "../../validations/validateData";
+import { signInSchema } from "../../validationSchemas/user";
+
+chai.use(chaiHttp);
+const expect = chai.expect;
```

Comment:
destructure expect `import { expect } from 'chai'`

Reply:
Destructure listen to @eokwukwe `const  { expect } = chai;`

Reply:
I told you to destructure this, do not resolve the comment if there is still a problem with the expression